libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -44,5 +62,56 @@ export class DelayedJob extends Model<DelayedJob> {
 
   @Column(BOOLEAN)
   isAcknowledged: boolean;
-  
+
+  @AllowNull
+  @Column(STRING)
+  name: string | null;
+
+  @AllowNull
+  @Column(STRING)
+  entityType: string | null;
+
+  @ForeignKey(() => Project)
+  @ForeignKey(() => Site)
+  @Column
+  entityId: number;
+
+  @BelongsTo(() => Project, {
+    foreignKey: "entityId",
+    constraints: false,
+    scope: {
+      entityType: "App\\Models\\V2\\Projects\\Project"
+    },
+    as: "entityProject"
```

Comment:
also removed. 
Removed the entityType too, because it was generating conflict with:
include: [
  { association: "entityProject" },
  { association: "entitySite" }
]

Reply:
Ah, I missed this comment when I did my most recent code review pass. This is definitely concerning. Can you share with me the details of the conflict and/or the DB query that's being issued? I'm doing something very similar in the most recent iteration of the entity service. It's a little difficult to follow, but [this statement](https://github.com/wri/terramatch-microservices/blob/release/utltimate-ulmus/apps/entity-service/src/trees/tree.service.ts#L73), in the case of a `Site`, is effectively creating two included associations using the same pattern I was outlining here (both joins against a single table, but with different scopes).